ladybug_comfort/cli/_helper.py

```diff
@@ -170,7 +171,10 @@ def thermal_map_csv(folder, temperature, condition, condition_intensity):
         'condition': os.path.join(folder, 'condition.csv'),
         'condition_intensity': os.path.join(folder, 'condition_intensity.csv')
     }
-    _data_to_csv(temperature, result_file_dict['temperature'])
-    _data_to_csv(condition, result_file_dict['condition'])
-    _data_to_csv(condition_intensity, result_file_dict['condition_intensity'])
+    with open(result_file_dict['temperature'], 'wb') as fp:
```

Comment:
I'm imagining that you would pass the boolean flag through this function, which will either write the results as plain text CSV or save them as numpy.